ddtrace/span.py

```diff
@@ -38,8 +38,10 @@
     from .tracer import Tracer
 
 
-_MetaKeyType = Union[Text, bytes]
-_MetaDictType = Dict[_MetaKeyType, Text]
+_TagNameType = Union[Text, bytes]
+_MetaDictType = Dict[_TagNameType, Text]
+_MetricsValueType = Union[int, float]
```

Comment:
do you want to have `long` too on Py2?
That should be `six.integer_types` then.